diff --git a/exec/cmap.c b/exec/cmap.c
index 9bd7e78..c73b036 100644
--- a/exec/cmap.c
+++ b/exec/cmap.c
@@ -409,6 +409,10 @@ static void cmap_sync_activate (void)
 	}
 
 
+	log_printf(LOGSYS_LEVEL_NOTICE,
+		"Highest config version (%"PRIu64") and my config version (%"PRIu64")",
+		cmap_highest_config_version_received, cmap_my_config_version);
+
 	if (cmap_highest_config_version_received != cmap_my_config_version) {
 		log_printf(LOGSYS_LEVEL_ERROR,
 		    "Received config version (%"PRIu64") is different than my config version (%"PRIu64")! Exiting",
